### DISCUSSION TITLE
Support sanitising data uris

### DISCUSSION THREAD

**Discussion by sahil-seth** [renovatebot/renovate#31719] [Suggest an Idea]
### Tell us more.

Currently, Renovate supports sanitising URLs in `lib/logger/utils.ts` but it does not cover sanitising [data uris](https://en.wikipedia.org/wiki/Data_URI_scheme#:~:text=The%20data%20URI%20scheme%20is,file%20literal%20or%20here%20document.)

We can potentially extend the `sanitizeUrls` function with a small change to support the use case.

Example for new use case:

`data:text/vnd-example;foo=bar;base64,R0lGODdh`

 I can push a PR for this as well.

**Comment by rarkins** [comment]
I don't fully understand this requirement, but if it's easier for you to push a PR to demonstrate it rather than keep describing it then please go ahead. e.g. is Renovate sanitizing a known secret in the URI, or sanitizing a particular section which can contain secrets?

**Reply by sahil-seth** [reply]
Created the PR here: https://github.com/renovatebot/renovate/pull/31721
